modules/reference/pages/admin/restart.adoc

```diff
@@ -1,9 +1,6 @@
 [[s3-sattools-config-restart]]
 = Restart
 
-The menu:Manager Configuration[Restart] page comprises the final step in configuring {productname}.
-
-image::admin_configuration_restart.png[scaledwidth=80%]
-
-Click the btn:[Restart] button to restart {productname} and incorporate all of the configuration options added on the previous screens.
-It will take between four and five minutes for a restart to finish.
+The menu:Admin[Manager Configuration > Restart] section allows you to restart {productname}.
+You will need to do this after making configuration changes.
+It will take four to five minutes for {productname} to become available again after a restart.
```

Comment:
Wondering whether we can take the "four to five minutes" duration for granted?

Reply:
Yeah, I wondered about that too. Perhaps we should say something like "some time" instead?